### PR TITLE
Update Tool Labs section to bring all of wmflabs to HTTPS

### PR DESCRIPTION
**Pull request by jonkolbert**

### CODE REVIEW

**Comment by jonkolbert** [comment]
@Hainish any idea why this isn't working?

File: src/chrome/content/rules/Wikimedia.xml [hunk 1]
```diff
@@ -64,10 +67,11 @@
 		<!-- mismatch on tools.wikimedia.de -->
 		<exclusion pattern="^http://tools\.wikimedia\.de/" />
 			<test url="http://tools.wikimedia.de/" />
+		<exclusion pattern="^http://civicrm\.wikimedia\.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Wikimedia.xml]: That's because of an incomplete cert chain. Could you document that?

Comment (jonkolbert) [reply src/chrome/content/rules/Wikimedia.xml]: @J0WI Done, and the build finally passed.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**